dsc/tests/dsc_discovery.tests.ps1

```diff
@@ -77,13 +77,8 @@ Describe 'tests for resource discovery' {
             "get": {
                 "executable": "dsctest",
                 "args": [
-                    "echo",
-                    "--input",
-                    "{json}"
-                ],
-                "input": {
-                    "arg": "{json}"
-                }
+                    "echo"
```

Comment:
Why it's not matching the
```
{
  "jsonInputArg": "<parameter>",
  "mandatory": true
}
```
syntax?

Reply:
This is just a test resource that doesn't get run to validate the semver.  Maybe I'll rename the type to make this clear and remove the args since they aren't used at all.